example/src/main/java/com/iopipe/examples/Hello.java

```diff
@@ -17,8 +18,17 @@
 	 * @since 2017/12/13
 	 */
 	@Override
-	protected final String wrappedHandleRequest(String __input, Context __c)
+	protected final String wrappedHandleRequest(IOpipeExecution __exec,
+		String __input)
 	{
+		// Send a message to the example plugin
+		__exec.<ExampleExecution>plugin(ExampleExecution.class, (__s) ->
```

Comment:
Is there an example using custom metrics?

Reply:
Yes, there is an example.